src/CoreWCF.Primitives/src/CoreWCF/Description/MessagePartDescription.cs

```diff
@@ -90,7 +91,17 @@ public bool Multiple
             get { return multiple; }
             set { multiple = value; }
         }
-
+        public ProtectionLevel ProtectionLevel
+        {
+            get { return this.protectionLevel; }
+            set
+            {
+              //  if (!ProtectionLevelHelper.IsDefined(value))
+              //      throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new ArgumentOutOfRangeException("value"));
```

Comment:
This is to prevent people doing something like MessagePartDescription.ProtectionLevel = (ProtectionLevel)137. We do have ProtectionLevelHelper already